pkg/rpc/server/server_test.go

```diff
@@ -867,16 +919,26 @@ func testRPCProtocol(t *testing.T, doRPCCall func(string, string, *testing.T) []
 			checkErrGetResult(t, body, true)
 		})
 
-		wif := testchain.WIF(0)
-		acc, err := wallet.NewAccountFromWIF(wif)
+		priv0 := testchain.PrivateKeyByID(0)
+		acc0, err := wallet.NewAccountFromWIF(priv0.WIF())
 		require.NoError(t, err)
+
+		addNetworkFee := func(tx *transaction.Transaction) {
+			size := io.GetVarSize(tx)
+			netFee, sizeDelta := core.CalculateNetworkFee(acc0.Contract.Script)
+			tx.NetworkFee += netFee
+			size += sizeDelta
+			tx.NetworkFee = tx.NetworkFee.Add(util.Fixed8FromFloat(float64(size) * chain.FeePerByte().FloatValue()))
```

Comment:
I've seen this code somewhere already.

Reply:
That's a small part from RPC client, but it differs a bit, because we have `chain` here and don't need to make a separate RPC-request to get FeePerByte.

Reply:
Also removed floats.